src/components/AppContainer.jsx

```diff
@@ -0,0 +1,18 @@
+import React, { Component, Fragment } from "react";
+import HeaderApp from "./Header";
+import FooterApp from "./Footer";
+import "../styles/AppContainer.css";
```

Comment:
Keep styles with the component.

For every component is more recommended to use the following structure
```
SomeComponent/
  SomeComponent.js
  index.js
  SomeComponent.css
  SomeComponent.test.js
```
https://medium.com/@alexmngn/how-to-better-organize-your-react-applications-2fd3ea1920f1

Note,

in `index.js` usually exported the main needed stuff from the component including the component itself.

`export * from './SomeComponent';`